docs/datasources.mdx

```diff
@@ -0,0 +1,103 @@
+  - `nutanix_insecure` (boolean) - Authorize connection on Prism Central without valid certificate
+  - `cpu` (integer) - Number of vCPU for temporary VM
+  - `memory_mb` (integer) - Size of vRAM for temporary VM (in megabytes)
+  - `cd_files` (array of strings) - A list of files to place onto a CD that is attached when the VM is booted. This can include either files or directories; any directories will be copied onto the CD recursively, preserving directory structure hierarchy
+  - `cd_label` (string) - Label of this CD Drive
+
+## Output configuration
+These parameters allow to configure everything around image creation, from the temporary VM connection to the final image definition.
+
+### All OS
+- `image_name` (string) - Name of the output image
+- `shutdown_command` (string) - Command line to shutdown your temporary VM
+- `shutdown_timeout` (string) - Timeout for VM shutdown (format : 2m)
+
+### Dedicated to Linux
+- `user_data` (string) - cloud-init (Linux) content base64 coded.  
+- `ssh_username` (string) - user for ssh connection initiated by Packer.
+- `ssh_password` (string) - password for the this ssh user.
+
+### Dedicated to Windows
+- `communicator` (string) - Protocol used for Packer connection (ex "winrm")
+- `winrm_port` (integer) - Port for WinRM communication (Default is 5986)
+- `winrm_insecure` (boolean) - Allow insecure connection on WinRM
+- `winrm_use_ssl` (boolean) - Request SSL connection with WinRM
+- `winrm_timeout` (string) - Timeout for WinRM (format 45m)
+- `winrm_username` (string) - User login for WinRM connection 
+- `winrm_password` (string) - Password this User
+
+
+## Disk configuration
+Use `vm_disks{}` entry to configure disk to your VM image. If you want to configure several disks, use this entry multiple times.
+
+All parameters of this `vm_disks` section are described below.
+
+2 types of disk configurations can by used : disk image and ISO image.
+
+### Disk image
+- `image_type` (string) - "DISK_IMAGE"
+- `source_image_name` (string) - Name of the image used as disk source (If you want an empty disk, do not specify source)
+- `disk_size_gb` (integer) - size of th disk (in gigabytes)
+
+Sample:
+```hcl
+  vm_disks {
+      image_type = "DISK_IMAGE"
+      source_image_name = "<myDiskImage>"
+      disk_size_gb = 40
+  }
+```
+### ISO Image
+- `image_type` (string) - "ISO_IMAGE"
+- `source_image_name` (string) - Name of the ISO image to mount
+
+Sample:
+```hcl
+  vm_disks {
+      image_type = "ISO_IMAGE"
+      source_image_name = "<myISOimage>"
+  }
+```
+
+## Network Configuration
+Use `vm_nics{}` entry to configure NICs in your image
+
+In this section, you have to define network you will connect on with this keyword :
+
+- `subnet_name` (string) - Name of the cluster subnet to use
+
+Sample
+```hcl
+  vm_nics {
+    subnet_name       = "<mySubnet>"
+  }
+```
+
+## Samples
+You can find samples [here](../example) for these instructions usage.
```

Comment:
put an absolute link here or it will potentially failed once imported in the Hashicorp doc side

Reply:
fixed in last commit